cdap-docs/reference-manual/source/release-notes.rst

```diff
@@ -24,6 +24,156 @@ Cask Data Application Platform Release Notes
    :depth: 2
 
 
+`Release 2.8.0 <http://docs.cask.co/cdap/2.8.0/index.html>`__
+=============================================================
+
+General
+-------
+
+- The :ref:`HTTP RESTful API v2 <http-restful-api-v2>` is deprecated, replaced with the
+  :ref:`namespaced HTTP RESTful API v3 <http-restful-api-v3>`.
+
+- Added log rotation for CDAP programs running in YARN containers
+  (`CDAP-1295 <https://issues.cask.co/browse/CDAP-1295>`__).
+
+- Added the ability to submit to non-default YARN queues to provide 
+  :ref:`resource guarantees <resource-guarantees>`
+  for CDAP Master Services, CDAP Programs, and Explore Queries
+  (`CDAP-1417 <https://issues.cask.co/browse/CDAP-1417>`__).
+
+- Added the ability to :ref:`prune invalid transactions <tx-maintenance>`
+  (`CDAP-1540 <https://issues.cask.co/browse/CDAP-1540>`__).
+
+- Added the ability to specify 
+  :ref:`custom logback file for CDAP programs <application-logback>`
+  (`CDAP-1741 <https://issues.cask.co/browse/CDAP-1741>`__).
+
+- System HTTP services now bind to all interfaces (0.0.0.0), rather than 127.0.0.1.
+
+New Features
+------------
+
+- **Command Line Interface (CLI)**
+
+  - Support for runtime arguments, which can be listed by running ``"cdap-cli.sh --help"``.
+  - Table rendering can be configured using ``"cli render as <alt|csv>"``. 
+    The option ``"alt"`` is the default, with ``"csv"`` available for copy & pasting.
+  - Stream statistics can be computed using ``"get stream-stats <stream-id>"``.
```

Comment:
There is also a new feature here that you can now directly connect to a CDAP of your choice at startup instead of starting the CLI and then issuing a ``connect`` command using ``cdap-cli.sh --uri <uri>`` /cc @jwang47